jax/_src/numpy/lax_numpy.py

```diff
@@ -3617,6 +3617,18 @@ class CClass(_AxisConcat):
 c_ = CClass()
 
 
+class S_class():
+  """Handy tool that returns `slice` objects from standard array indexing syntax.
+
+  Examples
+    >>> jnp.s_[1:2:10]
+    slice(1, 2, 10)
+  """
+  def __getitem__(self,slc):
+    return slc
+
+s_ = S_class()
+
```

Comment:
In this case, since the return value does not contain any arrays, it may be sufficient to write `s_ = np.s_` rather than re-implementing it. What do you think?

Reply:
That does sound easier. The issue discouraged using numpy code as it is, so I just implemented it all over again. I was wondering whether I missed I something because this function seemed rather easy but was still the last one left

Reply:
Sorry for the lack of clarity there: there is no problem with using numpy functionality via import in the JAX source; the problem is with copying numpy source code and pasting it into the JAX source code.

Reply:
Yep..it would be cleaner to just use the numpy implementation then!